### ISSUE TITLE
Windows updater fails due to EPERM: operation not permitted

### ISSUE THREAD

**Issue by kelsos** [rotki/rotki#4671]
## Problem Definition

Windows auto-update fails for users with `EPERM: operation not permitted, mkdir 'C:\Users\runneradmin'`

Now `runneradmin` is the user in the GitHub Actions that builds the application.

This should normally be `%USERPROFILE%` which would dynamically translate to the logged user's account, but it seems that somehow this gets interpolated during the build process and ends up with the `runneradmin` user hard-coded.

This results to the updater trying to download the update in the wrong user directory failing since it doesn't have permission to do so.

## Logs

```
2022-08-09T08:52:20.909Z: (info): Found version 1.25.1 (url: rotki-win32_x64-v1.25.1.exe)
2022-08-09T08:52:21.841Z: (info): Downloading update from rotki-win32_x64-v1.25.1.exe
2022-08-09T08:52:21.843Z: (debug): updater cache dir: C:\Users\runneradmin\AppData\Local\rotki-updater
2022-08-09T08:52:21.845Z: (error): Error: Error: EPERM: operation not permitted, mkdir 'C:\Users\runneradmin'
2022-08-09T08:52:21.846Z: Error: EPERM: operation not permitted, mkdir 'C:\Users\runneradmin'
```

### System Description

Operating system: Windows (Any)
Rotki version: v1.25.x

**Comment by kelsos** [comment]
I am not sure this is related to the issue above but on linux we have a different error

```
2022-08-10T15:49:26.587Z: (warn): APPIMAGE env is not defined, current application is not an AppImage
2022-08-10T15:49:26.910Z: Restarting the backend
2022-08-10T15:49:26.922Z: Determining if we are packaged by seeing if /tmp/.mount_rotki-aB9qEs/resources/backend exists
2022-08-10T15:49:26.923Z: Starting packaged python subprocess: /tmp/.mount_rotki-aB9qEs/resources/backend/rotki-core-1.25.0-linux --rest-api-port 4242 --loglevel critical --api-cors app://* --logfile /home/user/.config/rotki/logs/rotkehlchen.log
2022-08-10T15:49:26.936Z: The Python sub-process started on port: 4242 (PID: 42475)
2022-08-10T15:49:43.608Z: (error): Error: Error: Please check update first
    at hH.downloadUpdate (/tmp/.mount_rotki-aB9qEs/resources/app.asar/dist/background.js:73:12732)
    at IpcMainImpl.<anonymous> (/tmp/.mount_rotki-aB9qEs/resources/app.asar/dist/background.js:77:73435)
    at IpcMainImpl.emit (node:events:390:28)
    at EventEmitter.<anonymous> (node:electron/js2c/browser_init:161:10780)
    at EventEmitter.emit (node:events:390:28)
2022-08-10T15:49:43.609Z: Error: Please check update first
```

**Comment by LefterisJP** [comment]
Should be fixed by #4696
